subgraph/schema.graphql

```diff
@@ -0,0 +1,112 @@
+type Config @entity {
+  id: ID!
+  conviction: ConvictionConfig
+  voting: VotingConfig
+}
+
+type ConvictionConfig @entity {
+  id: ID!
+  decay: BigInt!
+  weight: BigInt!
+  maxRatio: BigInt!
+  pctBase: BigInt!
+  stakeToken: Token
+  totalStaked: BigInt!
+  maxStakedProposals: Int!
+  minThresholdStakePercentage: BigInt!
+  requestToken: Token
+}
+
+type VotingConfig @entity {
+    id: ID!
+    token: Token!
+    supportRequiredPct: BigInt!
+    minAcceptQuorumPct: BigInt!
+    durationBlocks: BigInt!
+    bufferBlocks: BigInt!
+    executionDelayBlocks: BigInt!
+}
+
+type Proposal @entity {
+  id: ID!
+  number: BigInt!
+  creator: Bytes!
+  status: ProposalStatus!
+  type: ProposalType!
+  createdAt: BigInt!
+  
+  # Proposal data
+  name: String
+  link: String
+  stakes: [Stake!] @derviedFrom(field: "proposal")
+  beneficiary: Bytes
+  requestedAmount: BigInt
+  totalTokensStaked: BigInt
+
+  # Voting data
+  metadata: String
+  startBlock: BigInt
+  executionBlock: BigInt
+  snapshotBlock: BigInt
+  supportRequiredPct: BigInt
+  minAcceptQuorum: BigInt
+  yea: BigInt
+  nay: BigInt
+  votingPower: BigInt
+  script: Bytes
+  casts: [Cast!] @derivedFrom(field: "proposal")
+}
+
+type Cast @entity {
+  id: ID!
+  voter: Bytes!
+  supports: Boolean!
+  voterStake: BigInt!
+  proposal: Proposal!
+  createdAt: BigInt!
+}
+
+type Stake @entity {
+  id: ID!
+  type: StakeType!
+  entity: Bytes!
+  proposal: Proposal!
+  amount: BigInt!
+  createdAt: BigInt!
+}
+
+type StakeHistory @entity {
+  id: ID!
+  type: StakeType!
+  entity: Bytes!
+  proposal: Proposal!
+  tokensStaked: BigInt!
+  totalTokensStaked: BigInt!
+  conviction: BigInt!
+  time: BigInt!      # Block at which was created
```

Comment:
i think that at some point we might want to change this naming hehe

Reply:
haha yeah maybe we could change it now actually (don't think will be a huge work updating the frontend), will do in the connector PR